src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -470,7 +470,10 @@ export type Config = {
     upgradeAvailable: boolean;
     latest: string;
   };
-  statuspage: string | null;
+  statuspage?: {
```

Comment:
@EvanPurkhiser what's this change about? Don't see any code changes related to this

Reply:
This was wrong, check `client_config.py`